### PR TITLE
Improve translate and mend default value

### PR DESCRIPTION
**Pull request by KuwaLee**

### CODE REVIEW

File: shared/src/commonMain/resources/assets/values-tr-TR/strings.xml
Comment (toasterofbread) [inline shared/src/commonMain/resources/assets/values-tr-TR/strings.xml:1]: None of these changes are needed, '$splitand' is already correct

File: shared/src/commonMain/resources/assets/values-zh-TW/strings.xml
Comment (toasterofbread) [inline shared/src/commonMain/resources/assets/values-zh-TW/strings.xml:1]: '$split 與' should be '$split與'

File: shared/src/commonMain/resources/assets/values/strings.xml
Comment (toasterofbread) [inline shared/src/commonMain/resources/assets/values/strings.xml:1]: '$splitand' is already correct

**Review by toasterofbread — CHANGES_REQUESTED** [review]

File: shared/src/commonMain/resources/assets/values-en-GB/strings.xml
[the remark this answers is not in the document]
Comment (KuwaLee) [reply shared/src/commonMain/resources/assets/values-en-GB/strings.xml:1]: I tested that if this file does not exist en-GB won't take effect
I have no idea why

[the remark this answers is not in the document]
Comment (toasterofbread) [reply shared/src/commonMain/resources/assets/values-en-GB/strings.xml:1]: Ah, OK

**Comment by KuwaLee** [comment]
Thanks for your patience to review my PR(so frequently), I'm busy today, I will provide ytm-kt translation ASAP
BTW, [getHoursMinutesSecondsSuffixes.kt](https://github.com/toasterofbread/ytm-kt/blob/d91e07c94699df76aa7d41219543b3930c9cc435/library/src/commonMain/kotlin/dev/toastbits/ytmkt/uistrings/localised/getHoursMinutesSecondsSuffixes.kt) is not full id, is it possible for translation?

**Comment by toasterofbread** [comment]
No worries, thanks for the contributions!

> BTW, [getHoursMinutesSecondsSuffixes.kt](https://github.com/toasterofbread/ytm-kt/blob/d91e07c94699df76aa7d41219543b3930c9cc435/library/src/commonMain/kotlin/dev/toastbits/ytmkt/uistrings/localised/getHoursMinutesSecondsSuffixes.kt) is not full id, is it possible for translation?

Yes, just change it to this:

```
fun getHoursMinutesSecondsSuffixes(hl: String?): HMSData? {
    when (hl) {
        "zh-CN" -> return HMSData("时间", "分", "秒")
        // Add full ID implementations here (don't remove zh from the section below though)
    }
    
    return when (hl?.split('-', limit = 2)?.firstOrNull()) {
        "en", null -> HMSData("hours", "minutes", "seconds", " ")
        "ja" -> HMSData("時間", "分", "秒")
        "zh" -> HMSData("时间", "分", "秒")
        "fr" -> HMSData("heures", "minutes", "secondes", " ")
        "tr" -> HMSData("saat", "dakika", "saniye", " ")
        "ru" -> HMSData("часы", "минуты", "секунды", " ")
        else -> null
    }
}
```

**Pull request merged**